fuzz/fuzz_targets/mt/fuzz_extension_attack.rs

```diff
@@ -0,0 +1,51 @@
+#![no_main]
+
+use ark_bls12_377::Fr as Fr377;
+use ark_bls12_381::Fr as Fr381;
+use ark_bn254::Fr as Fr254;
+use libfuzzer_sys::fuzz_target;
+use arbitrary::{Arbitrary, Unstructured};
+use jf_rescue::{crhf::RescueCRHF, RescueParameter};
+use jf_merkle_tree::{prelude::RescueMerkleTree, MerkleTreeScheme};
+use ark_ff::{Field, PrimeField, Zero};
+
+
+#[derive(Debug, Arbitrary)]
+struct ExtensionAttackInput {
+    forged_val: u64,
+    attack_pos: u64,
+}
+
+fuzz_target!(|data: &[u8]| {
+    let mut u = Unstructured::new(data);
+
+    test_extension_attack_helper::<Fr254>(&mut u);
+    test_extension_attack_helper::<Fr377>(&mut u);
+    test_extension_attack_helper::<Fr381>(&mut u);
+});
+
+fn test_extension_attack_helper<F: RescueParameter>(u: &mut Unstructured) {
+    if let Ok(input) = ExtensionAttackInput::arbitrary(&mut u) {
+        let forged_val = F::from(input.forged_val);
+        let attack_pos = input.attack_pos;
+        let forged_pos = attack_pos * 3 + 2;
```

Comment:
instead of using hardcoded value, let's either add explanatory code doc on top of this whole function or name `const arity = 3` 

especially explain the `digest_leaf` computation on `H(0 || pos || val)` where `0` is leaf_node_domain_separator... to help readers understand why `+2`